src/Service/CloudFormation/src/Result/StackEvent.php

```diff
@@ -72,7 +72,7 @@ class StackEvent
      *   PhysicalResourceId: ?string,
      *   ResourceType: ?string,
      *   Timestamp: \DateTimeInterface,
-     *   ResourceStatus: null|\AsyncAws\CloudFormation\Enum\ResourceStatus::CREATE_IN_PROGRESS|\AsyncAws\CloudFormation\Enum\ResourceStatus::CREATE_FAILED|\AsyncAws\CloudFormation\Enum\ResourceStatus::CREATE_COMPLETE|\AsyncAws\CloudFormation\Enum\ResourceStatus::DELETE_IN_PROGRESS|\AsyncAws\CloudFormation\Enum\ResourceStatus::DELETE_FAILED|\AsyncAws\CloudFormation\Enum\ResourceStatus::DELETE_COMPLETE|\AsyncAws\CloudFormation\Enum\ResourceStatus::DELETE_SKIPPED|\AsyncAws\CloudFormation\Enum\ResourceStatus::UPDATE_IN_PROGRESS|\AsyncAws\CloudFormation\Enum\ResourceStatus::UPDATE_FAILED|\AsyncAws\CloudFormation\Enum\ResourceStatus::UPDATE_COMPLETE|\AsyncAws\CloudFormation\Enum\ResourceStatus::IMPORT_FAILED|\AsyncAws\CloudFormation\Enum\ResourceStatus::IMPORT_COMPLETE|\AsyncAws\CloudFormation\Enum\ResourceStatus::IMPORT_IN_PROGRESS|\AsyncAws\CloudFormation\Enum\ResourceStatus::IMPORT_ROLLBACK_IN_PROGRESS|\AsyncAws\CloudFormation\Enum\ResourceStatus::IMPORT_ROLLBACK_FAILED|\AsyncAws\CloudFormation\Enum\ResourceStatus::IMPORT_ROLLBACK_COMPLETE,
+     *   ResourceStatus: ?\AsyncAws\CloudFormation\Enum\ResourceStatus::*,
```

Comment:
I hesitate to siwtch `?Foo` to `null|lFoo` for consistency

Reply:
`null|Foo` makes more sense.